add(rse_basics): Add argument why RSE is relevant in industry (#110)
Simulation-Software-Engineering/Lecture-Material: 00_organization/rse_basics_slides.md

- Companies use (more and more) the same workflows and tools.
    - It is not just about coding. It is about collaborative work.
- Open-source development excellent door opener for industry.

> - Many companies use open-source software tools and frameworks and also need to make contributions.

Reviewer: ```suggestion
- Some companies use open-source software and need to make contributions.
```

I am not completely sure whether *many* need to make contributions.
